pkg/services/services.go

```diff
@@ -241,6 +241,8 @@ func generateKubeletArgs(argsFromIgnition map[string]string, debug bool) ([]stri
 		// Windows host.
 		// See: https://kubernetes.io/docs/concepts/configuration/windows-resource-management/#resource-management-cpu
 		"--windows-priorityclass=" + windowsPriorityClass,
+		"--image-credential-provider-bin-dir=" + windows.K8sDir,
```

Comment:
consider using the KubeletConfiguration instead

https://github.com/openshift/windows-machine-config-operator/blob/6ef54058a437967ca61755c31631898679a83a20/pkg/nodeconfig/nodeconfig.go#L662

Reply:
I was not able to find this flag in that struct